changelog.md

```diff
@@ -0,0 +1,16 @@
+### x
+
+* Travis CI
+* PHPUnit testing
```

Comment:
I've also added changelog to make it easier for developers to see changes between new versions.

Changes yet not tagged are in version x.